package.json

```diff
@@ -12,23 +12,23 @@
   },
   "files": [
     "dist/"
-  ],  
+  ],
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/ancashoria/graphql-kafka-subscriptions.git"
+    "url": "git+https://github.com/aerogear/graphql-kafka-subscriptions.git"
   },
   "keywords": [
     "graphql",
     "kafka",
     "apollo",
     "subscriptions"
   ],
-  "author": "Ancas Horia",
+  "author": "Enda Phelan",
```

Comment:
:D

Reply:
I am not sure what the usual procedure is here, should we keep original?

Reply:
I always put Wojciech Trocki by default :-)

Reply:
ROFL - do not matter for me actually